### DISCUSSION TITLE
v1.5.5-beta

### DISCUSSION THREAD

**Discussion by albertpb** [albertpb/sd-manager#19] [General]
- Fix filter making to reimport all images again
- Add option none to tag
- Fix not auto tagging new images when import

<hr /><em>This discussion was created from the release <a href='https://github.com/albertpb/sd-manager/releases/tag/v1.5.5'>v1.5.5-beta</a>.</em>

**Comment by AsuraAce** [comment]
do we really want to delete tags from the image itself per default? i think this option should only remove the tag itself but not from the image, in case we change our mind at some point we could re-add those tags again. 

![image](https://github.com/albertpb/sd-manager/assets/13394741/e44d3599-cfb7-4c5d-a76c-8be7ec197d59)

**Reply by albertpb** [reply]
The tag itself has an id to make the relationship with the image. That's why deleting the tag will remove this relationship as well.

**Comment by AsuraAce** [comment]
you could hide the tag instead of deleting it. just a suggestion though, if it you want to keep this way that´s fine as well.
